app/assets/stylesheets/dashboard/service_calendar.sass

```diff
@@ -1,3 +1,14 @@
+.user-view-ssr-modal
+  width: 95%
+
+.visit-before-container, .visit-day-container, .visit-after-container
```

Comment:
That's exactly what I'm doing, but I suppose it really is a game of whack-a-mole w.r.t. to styling and views/stylesheets. If you guys want the moles in the view, then I'll put them back.